config.js

```diff
@@ -74,8 +70,6 @@ module.exports = {
 		]
 	},
 
-	cssmin: {},
```

Comment:
cssmin is still in use. why did you delete this?

Reply:
I don't think cssmin is still in use. I could not find any reference to it.
This got replaced by gulp-clean-css since it was deprecated.

Reply:
Right, sorry, I mixed it up with the cleanCss option